ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Address.java

```diff
@@ -45,6 +45,9 @@
   // Last address that can be generated for a pre-compiled contract
   public static final Integer PRIVACY = Byte.MAX_VALUE - 1;
   public static final Address DEFAULT_PRIVACY = Address.precompiled(PRIVACY);
+  public static final Address ONCHAIN_PRIVACY = Address.precompiled(PRIVACY - 1);
```

Comment:
no - there can be 16^20 addresses. The reason we are limited to the first 256 is that we parse the address from a single byte.

I think this can be easily solved when it becomes a problem. No need to do it here.